tests/nextup.input

```diff
@@ -0,0 +1,19 @@
+# name nextup
+# arg1 decimal
+# ret  decimal
+
+0                                        0.000001 decimal32
+-0                                       0.000001 decimal32
+1                                        1.000001 decimal32
+2                                        2.000001 decimal32
+3                                        3.000001 decimal32
```

Comment:
I would also add a test for the smallest possible negative value, and -0.

Reply:
The existing test suite can also be used to substitute interesting values for each type (e.g infinite, min/max values).

Reply:
Now I'm using the test suite constants but I think some of them are wrong. For example, `DEC_MAX_EXP` is replaced by 90/369/6111 for 32/64/128 bits respectively and I think it should be 97/385/6145 based on ISO TR-24732.

There's negative 0 and smallest negative test for every type.

Reply:
The maximum exponent is dissimilar to BFP. The mantissa is always an integral value, thus the maximum exponent of the format is reduced depending on the precision of the format. The maximum representable exponent is a value with one significand leading the radix point e.g 1.000eX.

Reply:
Oh, now it makes sense. Thanks!